CMake modules get installed to a wrong path

When building the [ginkgo-hpc-git AUR package](https://aur.archlinux.org/packages/ginkgo-hpc-git), I noticed that it installs the following cmake files:
```
ginkgo-hpc-git /usr/lib/cmake/
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/GinkgoConfig.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/GinkgoConfigVersion.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/GinkgoTargets-release.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/GinkgoTargets.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/Modules/
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/Modules/CudaArchitectureSelector.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/Modules/FindHWLOC.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/Modules/FindNUMA.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/Modules/FindPAPI.cmake
ginkgo-hpc-git /usr/lib/cmake/Ginkgo/Modules/Modules/hwloc_helpers.cmake
```

The `Modules/Modules` part is most likely a bug which causes the following error when `find_package(Ginkgo)` is used in another project:
```
CMake Error at /usr/lib64/cmake/Ginkgo/GinkgoConfig.cmake:187 (find_package):
  By not providing "FindHWLOC.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "HWLOC", but
  CMake did not find one.

  Could not find a package configuration file provided by "HWLOC" with any of
  the following names:

    HWLOCConfig.cmake
    hwloc-config.cmake

  Add the installation prefix of "HWLOC" to CMAKE_PREFIX_PATH or set
  "HWLOC_DIR" to a directory containing one of the above files.  If "HWLOC"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  CMakeLists.txt:55 (find_package)


-- Configuring incomplete, errors occurred!
```

--- a/cmake/install_helpers.cmake
+++ b/cmake/install_helpers.cmake
@@ -105,7 +105,7 @@ function(ginkgo_install)
     endif()
 
     # Install CMake modules
-    install(DIRECTORY "${Ginkgo_SOURCE_DIR}/cmake/Modules"
+    install(DIRECTORY "${Ginkgo_SOURCE_DIR}/cmake/Modules/"
         DESTINATION "${GINKGO_INSTALL_MODULE_DIR}"
         FILES_MATCHING PATTERN "*.cmake"
         )

